### DISCUSSION TITLE
dynamic content for meta tags is not  working in next js

### DISCUSSION THREAD

**Discussion by harshakulkarni23** [vercel/next.js#14481] [Help]
This is my code. I want to show meta tags with dynamic content. But it is not working.
<Head> 
<meta httpEquiv="X-UA-Compatible" content="IE=edge,chrome=1" />
          <meta name="viewport" content="initial-scale=1" />
          <meta charSet="utf-8" />
          <meta property="og:site_name" content="Blogs" />
          <meta property="og:title" key="ogTitle" content={title} />
          <meta property="og:url" content="" />
          <meta property="og:type" content="blog" />
          <meta
            property="og:description"
            content={description}
          />
          <meta property="og:image" content={logo} />
          <meta property="og:image:width" content="299" />
          <meta property="og:image:height" content="314" />
          <meta itemProp="name" content={title} />
          <meta itemProp="url" content=""/>
          <meta
            itemProp="description"
            content={description}
          />
          <meta itemProp="image" content={logo} />
          <meta name="twitter:title" content={title} />
          <meta name="twitter:image" content={logo} />
          <meta name="twitter:url" content="" />
          <meta name="twitter:card" content="summary" />
          <meta
            name="twitter:description"
            content={description}
          />
          <meta
            name="description"
            content={description}
          />
</Head>

**Comment by IamLizu** [comment]
You may have forgotten to post your code.
